update.sh

```diff
@@ -49,6 +55,38 @@ for version in "${versions[@]}"; do
 		continue
 	fi
 
+	otpMajor="${otpMajors[$rcVersion]}"
+	otpVersion="$(
+		git ls-remote --tags https://github.com/erlang/otp.git \
+			"refs/tags/OTP-$otpMajor.*"\
+			| cut -d'/' -f3- \
+			| cut -d'^' -f1 \
+			| cut -d- -f2- \
+			| sort -uV \
+			| tail -1
+	)"
+	if [ -z "$otpVersion" ]; then
+		echo >&2 "warning: failed to get Erlang/OTP version for '$version' ($fullVersion); skipping"
+		continue
+	fi
+	# TODO these aren't published anywhere (nor is the tarball we download even provided by Erlang -- it's simply a "git archive" tar provided by GitHub)...
+	otpSourceSha256="$(wget -qO- "https://github.com/erlang/otp/archive/OTP-$otpVersion.tar.gz" | sha256sum | cut -d' ' -f1)"
```

Comment:
I really hate this bit (since it's soooo slow), but I don't really see a better solution ATM. :disappointed:

(The whole script still runs in ~17 seconds on my machine even doing this twice, so I think it's acceptable for now.)

Reply:
Feel free to complain on erlang-users, I will join you :) Erlang's patch release strategy makes some sense if you consider that it is a rare system where applying patches to only parts of the standard library is possible and used to be common. These days people deploy VM images, containers, and so on, and usually expect the entire runtime to be rebuilt and available 😞

Reply:
Using the GitHub API is OK, but is subject to per-IP rate limits, so isn't really tenable for our automated systems (which is why we use `git ls-remote --tags` instead, since it's not subject to the API limits). :disappointed:

Getting the tag isn't something I'm concerned terribly about -- we use `git ls-remote` very successfully in a lot of places; my concern is all around getting a checksum of the release, and I suppose complaining on erlang-users is the best we can do. :sweat_smile:

Reply:
OK, I will action [what I've discussed on erlang-questions with Lukas](http://erlang.org/pipermail/erlang-questions/2019-January/097067.html) next week, we'll have the GPG signatures in place not before long :shipit: 

cc @garazdawi

Reply:
This is still on my radar, I plan to pick it up as soon as I'm through with the current RabbitMQ/Prometheus context.